gateway/1.6/modules/ROOT/pages/flex-view-api-metrics.adoc

```diff
@@ -56,4 +56,5 @@ You can view the data points collected for the last given period of time (such a
 == What's Next?
 
 * Create alerts using the metrics related to an API instance with the xref:flex-use-api-alerts.adoc[Alerts feature in API Manager] or with the xref:monitoring::basic-alerts.adoc[Basic Alerts feature in Anypoint Monitoring].
-* View built-in API dashboards in xref:monitoring::api-analytics-dashboard.adoc[Anypoint Monitoring] where you can see additional metrics, filter data by date, and export data into a CSV (Titanium subscriptions only).
+* View built-in API dashboards in xref:monitoring::api-analytics-dashboard.adoc[Anypoint Monitoring] where you can see additional metrics, filter data by date, and export data into a CSV 
```

Comment:
```suggestion
* View built-in API dashboards in xref:monitoring::api-analytics-dashboard.adoc[Anypoint Monitoring] where you can see additional metrics, filter data by date, and export data into a CSV.
```
Does this need a period for consistency with the other list items? Or perhaps the "availabilityParenthetical" partial terminates the sentence with a period?